.github/workflows/release-api.yaml

```
@@ -41,8 +45,12 @@ jobs:
           push: true
           cache-from: type=registry,ref=ghcr.io/${{ env.IMAGE_NAME }}:cache
           cache-to: type=inline
+          secrets:
+            TURBO_TOKEN: ${{ secrets.TURBO_TOKEN }}
```

actionlint (1.7.4)

expected scalar node for string value but found mapping node with "!!map" tag

(syntax-check)